apps/src/code-studio/components/AssetManager.jsx

```diff
@@ -52,26 +56,48 @@ export default class AssetManager extends React.Component {
     useFilesApi: PropTypes.bool,
     soundPlayer: PropTypes.object,
     disableAudioRecording: PropTypes.bool,
+    projectId: PropTypes.string,
+    levelName: PropTypes.string,
 
     // For logging purposes
     imagePicker: PropTypes.bool, // identifies if displayed by 'Manage Assets' flow
-    projectId: PropTypes.string,
     elementId: PropTypes.string
   };
 
   constructor(props) {
     super(props);
     this.state = {
       assets: null,
+      starterAssets: null,
       statusMessage: props.uploadsEnabled ? '' : errorUploadDisabled,
       recordingAudio: false,
       audioErrorType: AudioErrorType.NONE
     };
   }
 
   componentWillMount() {
```

Comment:
`componentWillMount` [is deprecated](https://reactjs.org/docs/react-component.html#mounting).  Maybe we should change this up while we're in here?

Reply:
good call! i'll make a follow-up PR for this